website/docs/reference/global-configs.md

```diff
@@ -3,49 +3,61 @@ title: "Global Configs"
 id: "global-configs"
 ---
 
-dbt supports several global runtime configs. Starting in v1.0, all these configs can be set in three places:
-- `config:` block in `profiles.yml`, sometimes called "user config"
-- environment variables, prefixed with `DBT_`
-- CLI flags, which immediately follow `dbt` and precede your subcommand
+### About Global Configs
 
-Notes:
-- The precedence order is always (1) CLI flag > (2) env var > (3) profile config
-- All boolean configs can be turned on via a CLI flag named `--this-config`, and turned off with a CLI flag named `--no-this-config`
-- Use the profile config to set defaults for all projects running on your local machine
+Global runtime configurations (global configs) enable you to set default configuration values for all projects that you run locally on your computer. dbt supports several global configs.
 
-<File name='profiles.yml'>
+Starting in v1.0, you can set global configs in three places. When all three are set, command line flags take precedence, then environment variables, and last profile configs.
 
-```yaml
+#### Command line flags
 
-config:
-  this_config: true
+Command line (CLI) flags immediately follow `dbt` and precede your subcommand. When set, CLI flags override environment variables and profile configs. You can use `--this-config` CLI flag to turn on boolean configs, and a `--no-this-config` CLI flag to turn off boolean configs:
+
+<File name='CLI flags'>
+
+```text
+$ dbt --this-config
```

Comment:
@jtcohen6 is this example missing a subcomman? Or does "this-config"  represent an example config?

Reply:
Good call, it should probably be something like:
```
dbt --<this-config> <subcommand>
```